src/components/app-player/script.js

```diff
@@ -1,8 +1,158 @@
+import api from '@/api/index';
```

Comment:
Please update import, index is the default entry point
`import api from '@/api`

Reply:
Is not necessary now.